### PR TITLE
BaSP-TG-107: Update tasks form

### PR DESCRIPTION
**Pull request by AlejandroSantini**

### CODE REVIEW

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
Great job, just some little things.

**Review by Gearyandres94 — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]
🚀

File: src/components/Task/index.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const TaskForm = () => {
     description: Joi.string().min(3).max(50).required(),
     workedHours: Joi.number().min(1).required(),
     projectId: Joi.string().required(),
-    date: Joi.date().required()
+    date: Joi.date().min(Date()).required()
```
Comment (Gearyandres94) [inline src/components/Task/index.js:38]: This doesnt work when i choose today as date.

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]

File: src/components/Task/index.js (hunk 1 above)
Comment (Gearyandres94) [reply src/components/Task/index.js:38]: Capture

Comment (Gearyandres94) [reply src/components/Task/index.js:38]: Use this instead 
    date: Joi.date().greater(Date.now() - 24 * 60 * 60 * 1000).required(),

**Review by Gearyandres94 — DISMISSED** [review]
:rocket:

**Review by GuidoCerioni — APPROVED** [review]

**Review by Gearyandres94 — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**